cli/command/service/list.go

```diff
@@ -116,7 +118,7 @@ func printTable(out io.Writer, services []swarm.Service, running map[string]int)
 		if service.Spec.Mode.Replicated != nil && service.Spec.Mode.Replicated.Replicas != nil {
 			replicas = fmt.Sprintf("%d/%d", running[service.ID], *service.Spec.Mode.Replicated.Replicas)
 		} else if service.Spec.Mode.Global != nil {
-			replicas = "global"
+			replicas = fmt.Sprintf("%d/%d global", running[service.ID], tasksPerService[service.ID])
```

Comment:
Is the "total" here always using the total node count? If so, that should probably be changed to take constraints into account (which _is_ supported for global services)